lib/kafo/hiera_configurer.rb

```diff
@@ -1,111 +1,37 @@
-require 'fileutils'
-require 'tmpdir'
-
 module Kafo
   class HieraConfigurer
-    HIERARCHY_NAME = 'Kafo Answers'
-    HIERARCHY_FILENAME = 'kafo_answers.yaml'
-
-    attr_reader :temp_dir, :config_path, :data_dir, :logger
-
-    def initialize(user_config_path, modules, modules_order)
-      @user_config_path = user_config_path
-      @modules = modules
-      @modules_order = modules_order
-      @logger = KafoConfigure.logger
+    def self.default_config
+      {
+        'version' => 5,
+        'hierarchy' => [
+          {
+            'name' => 'Kafo Answers',
+            'path' => '%{facts.kafo.scenario.answer_file}',
+            'data_hash' => 'yaml_data',
+          },
+        ],
+      }
     end
 
-    def write_configs
-      build_temp_dir
-
-      if @user_config_path
-        logger.debug("Merging existing Hiera config file from #{@user_config_path}")
-        user_config = YAML.load(File.read(@user_config_path))
-      else
-        user_config = {}
-      end
-      logger.debug("Writing Hiera config file to #{config_path}")
-      File.open(config_path, 'w') do |f|
-        # merge required config changes into the user's Hiera config
-        f.write(format_yaml_symbols(generate_config(user_config).to_yaml))
-      end
-
-      logger.debug("Creating Hiera data files in #{data_dir}")
-      FileUtils.mkdir(data_dir)
-
-      File.open(File.join(data_dir, HIERARCHY_FILENAME), 'w') do |f|
-        f.write(format_yaml_symbols(generate_data(@modules).to_yaml))
-      end
+    def self.write_default_config(path)
+      File.open(path, 'w') { |f| f.write(YAML.dump(default_config)) }
```

Comment:
log with file path would be great

Reply:
Also added the log line in ExecutionEnvironment.